roles/openshift_descheduler/templates/descheduler-cronjob.yaml.j2

```diff
@@ -21,6 +21,9 @@ spec:
             - /bin/descheduler
             args:
             - --policy-config-file=/policy-dir/policy.yaml
+{% if openshift_descheduler_debug_level is defined and openshift_descheduler_debug_level is not none %}
```

Comment:
If the template is always used within the `openshift_descheduler` role (which should be the case everytime), the `openshift_descheduler_debug_level is defined` part is not needed.

Reply:
sounds good.